gcloud/bigtable/row.py

```diff
@@ -870,7 +870,7 @@ class _RegexFilter(RowFilter):
     """
 
     def __init__(self, regex):
-        self.regex = regex
+        self.regex = _to_bytes(regex)
```

Comment:
Do the protobufs for regex filters require UTF8-encoded bytes?  If so, would it be clearer to encode them inside `to_pb`?